datalake/src/main/java/com/sequenceiq/datalake/service/sdx/CloudStorageManifester.java

```diff
@@ -72,8 +74,21 @@ public CloudStorageRequest initCloudStorageRequest(DetailedEnvironmentResponse e
         return cloudStorageRequest;
     }
 
-    public CloudStorageRequest initSdxCloudStorageRequest(String cloudPlatform,
-            String blueprint, String clusterName, SdxCloudStorageRequest cloudStorage) {
+    public CloudStorageRequest initCloudStorageRequestFromExistingSdxCluster(DetailedEnvironmentResponse environment,
+            ClusterV4Response clusterV4Response, SdxCluster sdxCluster) {
+        boolean anyCloudStorageIsConfigured = !Strings.isNullOrEmpty(sdxCluster.getCloudStorageBaseLocation());
+        CloudStorageRequest cloudStorageRequest = new CloudStorageRequest();
```

Comment:
we can create a method in CloudStorageBase to copy base parameters from one CloudStorageBase to another CloudStorageBase. As I see request and response are the same so it would be enoght to call this "copy" method to copy all parameters.